source/sass/buyers-guide/bg-main.scss

```diff
@@ -65,6 +65,12 @@ html {
   background: $pni-product-image-background;
 }
 
+.bg-call-to-action-content,
+.bg-call-to-action-content > .rich-text > p {
+  // Overriding the default styling for a paragraph.
```

Comment:
If you wanted an tailwind only solution, you could always use arbitrary variants! https://tailwindcss.com/blog/tailwindcss-v3-1#arbitrary-values-but-for-variants we would need to update tailwind but it would be an easy upgrade

Reply:
Ah awesome. I was looking for that but did not find it in the docs for some reason. I will update since this is much easier for this onetime use 👍